src/collective/taxonomy/utility.py

```diff
@@ -14,33 +15,34 @@
 from plone.dexterity.interfaces import IDexterityFTI
 from plone.memoize import ram
 
-from zope.component import getMultiAdapter
-from zope.component.hooks import getSite
-from zope.component.interfaces import ComponentLookupError
-from zope.interface import implements
+from zope.interface import implementer
 from zope.lifecycleevent import modified
 
 import generated
 import logging
 
 from copy import copy
 
-from collective.taxonomy import PATH_SEPARATOR
+from os import sep as PATH_SEPARATOR
```

Comment:
Here PATH_SEPARATOR is used for the title of the term "level1 / level2 / level3"
Your change doesn't make sense. On windows you will have "\" as separator.
Please keep "from collective.taxonomy import PATH_SEPARATOR"
I monkey patch this constant in my project to change the separator to '>' so to have "level1 > level2 > level3". The use case is described in the README.